mcs/mcs/ikvm.cs

```diff
@@ -249,7 +249,11 @@ static StaticLoader ()
 			sdk_directory.Add ("4", new string[] { "4.0-api", "v4.0.30319" });
 			sdk_directory.Add ("4.0", new string[] { "4.0-api", "v4.0.30319" });
 			sdk_directory.Add ("4.5", new string[] { "4.5-api", "v4.0.30319" });
-			sdk_directory.Add ("4.6", new string [] { "4.5", "net_4_x", "v4.0.30319" });
+			sdk_directory.Add ("4.5.1", new string[] { "4.5.1-api", "v4.0.30319" });
+			sdk_directory.Add ("4.5.2", new string[] { "4.5.2-api", "v4.0.30319" });
+			sdk_directory.Add ("4.6", new string[] { "4.6-api", "v4.0.30319" });
+			sdk_directory.Add ("4.6.1", new string[] { "4.6.1-api", "v4.0.30319" });
+			sdk_directory.Add ("4.6.2", new string [] { "4.5", "4.6.2-api", "net_4_x", "v4.0.30319" });
```

Comment:
That's where we store our "latest" assemblies in the install. IIRC some distros don't ship all the reference assemblies so we need something to fallback to. I changed it to check for `4.6.2-api` first.